ext/SpeedyWeatherCUDAExt/legendre.jl

```diff
@@ -0,0 +1,127 @@
+# convert i, j indices of a matrix (here 0-based l,m though...) to a single 1-based running index
+import SpeedyWeather.LowerTriangularMatrices: ij2k
+
+# range of the running indices lm in a l-column (degrees of spherical harmonics)
+# given the column index m (order of harmonics) 
+get_lm_range(m, lmax) = ij2k(m, m, lmax):ij2k(lmax, m, lmax)
+
+# (inverse) legendre transform kernel, called from _legendre!
+function phase_factor_kernel!(
```

Comment:
this change hasn't been pushed yet?

Reply:
Not sure what change you're referring to?

Reply:
Where it is actually called from?

Reply:
It's invoked by the `@cuda` macro, so lines 95 (and 109)

Reply:
The name is bad, I'll change it